components/generic/LinkGroup.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <figure>
+    <figcaption
+      class="text-uppercase font-weight-bold"
+      data-qa="footer caption"
```

Comment:
This template should be generic and not refer to footer anywhere. Any footer-specific classes or data-qa attributes should go in the parent component.